index.js

```diff
@@ -125,13 +125,25 @@ function sendToIngest(event, context, aimsC, collectedBatch, callback) {
                     return callback(null, resp);
                 })
                 .catch(exception =>{
-                    return callback(`Unable to send to Ingest ${exception}`);
+                    return ingestError(event, context, exception, callback);
                 });
         }
     });
 }
 
 
+function ingestError(event, context, error, callback) {
+    if (error && error.statusCode && error.statusCode == 400) {
+        var body = (error.options && error.options.body) ?
+            error.options.body : undefined;
+        console.error(`Unable to send to Ingest. Message: ${body}`)
```

Comment:
It will not work if `body=undefined`.
please use `console.error('Unable to send to Ingest. Message:', body)` instead. Plus `;`

Reply:
That is strange , I think it should work. I have verified it in node console. I will add `;` ;-)

Reply:
Yep, it works, sorry. But the output is different.
```
node -p "var a = undefined; console.log('Output', a);"
Output undefined
undefined
```

vs

```
node -p "var a = undefined; console.log(\`Output ${a}\`);"
Output 
undefined
```

Reply:
no prob

Reply:
I maybe misunderstood it now. Do you mean it works but the output is not as expected? 

I have just ran this locally and in aws lambda:
```
    var body = undefined;
    console.error(`Unable to send to Ingest. Message: ${body}`);
```

and the output is as we would expect `Unable to send to Ingest. Message: undefined`

The latter one from your example also works for me though the output is on two lines:
```
al-car-06175:cwe-collector tdosoudil$ node -p "var a = undefined; console.log(\`Output ${a}\`);"
Output
undefined
al-car-06175:cwe-collector tdosoudil$
```